ui/components/app/signature-request-siwe/signature-request-siwe.js

```diff
@@ -82,8 +92,45 @@ export default function SignatureRequestSIWE({
     [cancelPersonalMessage],
   );
 
+  const balanceInBaseAsset = new Numeric(
+    fromAccount.balance,
+    16,
+    EtherDenomination.WEI,
+  )
+    .toDenomination(EtherDenomination.ETH)
+    .toBase(10)
+    .round(6)
+    .toString();
+
+  const getNetworkName = () => {
```

Comment:
I see this function exists elsewhere in the codebase a few times.  Would it be possible to move it to a helper file so we aren't repeating as much?  You'd pass the helper function `provider` and `t`